src/PKSim.Infrastructure/Reporting/Markdown/Builders/ParametersMarkdownBuilder.cs

```diff
@@ -15,10 +18,14 @@ public ParametersMarkdownBuilder(IParameterToParameterElementMapper parameterEle
          _parameterElementMapper = parameterElementMapper;
       }
 
-      public override void Report(IEnumerable<IParameter> parameters, MarkdownTracker tracker, int indentationLevel = 0)
+      public override void Report(IEnumerable<IParameter> parameters, MarkdownTracker tracker, int indentationLevel)
       {
-         var table = parameters.Select(x => _parameterElementMapper.MapFrom(x));
-         tracker.Add(table.ToMarkdownTable());
+         var parameterElements = parameters.Select(x => _parameterElementMapper.MapFrom(x)).ToList();
+         if (!parameterElements.Any())
```

Comment:
Only exports table fit here is actually some parameters. Otherwise the table looks very weird